deeprank/generate/DataGenerator.py

```diff
@@ -299,29 +224,17 @@ def create_database(
                 # check if we have a decoy or native
                 # and find the reference
                 if mol_name == bare_mol_name:
-                    ref = cplx
+                    ref = mutant.pdb_path
                 else:
-                    if len(self.all_native) > 0:
-                        ref = list(
-                            filter(lambda x: ref_name in x, self.all_native))
-                        if len(ref) == 0:
-                            raise ValueError('Native not found')
-                        else:
-                            if len(ref) > 1:
-                                warnings.warn(
-                                    f'Multiple native reference found, here used {ref[0]}')
-                            ref = ref[0]
-                        if ref == '':
-                            ref = None
-                    else:
-                        ref = None
+                    ref = None
 
-                # crete a subgroup for the molecule
+                # create a subgroup for the molecule
                 molgrp = self.f5.require_group(mol_name)
```

Comment:
Here, the feature data is stored in a HDF5 group, named after the molecule in which the mutant occurs. But what if we have two mutants in the same molecule? I suggest we don't use the name of the molecule for the group, but a mutant-based hash instead. @rgayatri what do you think?

Reply:
Mutant-based hash is a good idea. While we would look at single point mutations alone, and not two mutations simultaneously, I think hashing is good. 
Is it possible to have mutant-based hash and a mol_name?

Reply:
1. We could leave the molecule name in and add the hash as a suffix to unsure uniqueness.
2. We could also make super-groups in the HDF5 files, representing the molecule name, and then place the mutant group within the molecule group. This will require more coding though.

Reply:
The first one is what I thought should work.
We can have the molecule name and the hash for a mutant.

Reply:
OK, I'll change this pull request to realize it..